blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScm.java

```diff
@@ -70,7 +66,7 @@
     private static final String USER_EMAIL_SCOPE = "user:email";
     private static final String USER_SCOPE = "user";
     private static final String REPO_SCOPE = "repo";
-    private static final String DOMAIN_NAME="github-domain";
+    private static final String DOMAIN_NAME="blueocean-github-domain";
```

Comment:
>Your code should therefore be able to fall back and handle the situation where the add domain fails and either attempt to add the credentials into the global domain or declare that you cannot proceed.

I think we would rather fail than creating in global scope. So add credential is then going to fail and will be reported as error to UI.